01_fib/choohy/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+
+	fmt.Fprintln(out, "Call Fibonacci numbers!")
+	fib(7)
+}
+
+func fib(n int) {
+
```

Comment:
Again here, little nit but I would remove the extra white space to be consistent across all functions.